pkg/ring/lifecycler_test.go

```diff
@@ -17,18 +18,6 @@ import (
 	"github.com/cortexproject/cortex/pkg/util/test"
 )
 
-type flushTransferer struct {
-	lifecycler *Lifecycler
-}
-
-func (f *flushTransferer) Flush() {}
-func (f *flushTransferer) TransferOut(ctx context.Context) error {
```

Comment:
Unless I'm missing anything, the logic of this `flushTransferer` was not used, so I've removed it in favour of the tiny `nopFlushTransferer`